packages/ui/src/use-clipboard/index.tsx

```diff
@@ -18,14 +18,15 @@ const copyToClipboard = (value: string) => {
   el.style.left = '-9999px';
   document.body.appendChild(el);
 
-  const selected = document.getSelection().rangeCount > 0 ? document.getSelection().getRangeAt(0) : false;
+  const curSelection = document.getSelection();
```

Comment:
We should maybe add in a typeof document !== 'undefined' here

Reply:
Hm, not sure on that one - If we wanted those kind of checks, I'd rather change the `tsconfig` to not use `lib: dom`. But not worth it right now, IMO